mlton/atoms/prim.fun

```diff
@@ -1217,13 +1223,27 @@ fun 'a checkApp (prim: 'a t,
          andalso equals (arg0', arg 0)
          andalso equals (arg1', arg 1)
          andalso equals (arg2', arg 2)
+      fun fourArgs (arg0', arg1', arg2', arg3') () =
+         4 = Vector.length args
+         andalso equals (arg0', arg 0)
+         andalso equals (arg1', arg 1)
+         andalso equals (arg2', arg 2)
+         andalso equals (arg3', arg 3)
       fun fiveArgs (arg0', arg1', arg2', arg3', arg4') () =
          5 = Vector.length args
          andalso equals (arg0', arg 0)
          andalso equals (arg1', arg 1)
          andalso equals (arg2', arg 2)
          andalso equals (arg3', arg 3)
          andalso equals (arg4', arg 4)
+      fun sixArgs (arg0', arg1', arg2', arg3', arg4', arg5') () =
```

Comment:
Unused?

Reply:
Yes; this was necessary in the ref version but not this one.